cla_public/static-src/javascripts/modules/currencyInputValidation.js

```diff
@@ -0,0 +1,42 @@
+// ($(this).val() && Number($(this).val()) > 0)
+
+function isValidAmount(input) {
+    var maxVal = 100000000
+    var minVal = 0
```

Comment:
should these be `const`?

Reply:
Var is used because the file that requires it `form-error.js` seems to be using ES5 as when I try to use `const`, JS does not bundle correctly.